ansible/roles/operator-pipeline/templates/openshift/triggers/operator-hosted-pipeline-trigger.yml

```diff
@@ -0,0 +1,138 @@
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: operator-hosted-pipeline-github-listener
+spec:
+  serviceAccountName: pipeline
+  triggers:
+    - name: github-listener
+      interceptors:
+        - cel:
+            filter: >-
+              (
+                header.match("X-GitHub-Event", "pull_request")
+                && body.action in ["opened", "reopened", "synchronize"]
+                && body.pull_request.base.ref == "main"
```

Comment:
For the preprod environments there is a common repo with three different [branches](https://github.com/redhat-openshift-ecosystem/redhat-marketplace-operators-preprod/branches): `dev`, `qa` and `stage`. `main`only exists on the prod repo. We need a way to change that here.